src/main/scala/codacy/plugins/test/runner/ToolRunner.scala

```diff
@@ -0,0 +1,118 @@
+package codacy.plugins.test.runner
+
+import better.files._
+import sys.process._
+import com.codacy.analysis.core.model._
+import play.api.libs.json.Json
+import com.codacy.plugins.api.results.Tool
+import com.codacy.plugins.api
+
+import java.nio.file.Path
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/gh/codacy/codacy-plugins-test/pullRequest?prid=5569494)